src/en/mangaclash/build.gradle

```diff
@@ -3,7 +3,7 @@ ext {
     extClass = '.MangaClash'
     themePkg = 'madara'
     baseUrl = 'https://mangaclash.com'
```

Comment:
need to change here too

Reply:
Done Change in [src/en/mangaclash/build.gradle](https://github.com/keiyoushi/extensions-source/pull/2568/files/4c58fe84e99d2d8bbb96913ec0822bbbad76acab#diff-fc16076743a7c9368aa3b82aac278cc1bb46b5d77a91cd48d785c4fc696a2c09)  

Close issue #2546 & #2537 


![image](https://github.com/keiyoushi/extensions-source/assets/147808278/7731cede-48c8-432f-b20b-4ac236059d70)